x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/rules/add_prepackaged_rules_route.ts

```diff
@@ -58,63 +49,46 @@ export const createAddPrepackedRulesRoute = (
         const rulesToInstall = getRulesToInstall(rulesFromFileSystem, prepackagedRules);
         const rulesToUpdate = getRulesToUpdate(rulesFromFileSystem, prepackagedRules);
 
-        const spaceIndex = getIndex(spacesClient.getSpaceId, config);
+        const { signalsIndex } = siemClient;
         if (rulesToInstall.length !== 0 || rulesToUpdate.length !== 0) {
-          const spaceIndexExists = await getIndexExists(
+          const signalsIndexExists = await getIndexExists(
             clusterClient.callAsCurrentUser,
-            spaceIndex
+            signalsIndex
           );
-          if (!spaceIndexExists) {
-            return headers
-              .response({
-                message: `Pre-packaged rules cannot be installed until the space index is created: ${spaceIndex}`,
-                status_code: 400,
-              })
-              .code(400);
+          if (!signalsIndexExists) {
+            return siemResponse.error({
+              statusCode: 400,
+              body: `Pre-packaged rules cannot be installed until the signals index is created: ${signalsIndex}`,
```

Comment:
Our one integration test failure is a consequence of this copy change, here. I can revert it but I thought it might be a good opportunity to discuss the messaging we present to the client: "space index" vs "signals index." It looks like we use both interchangeably right now, but consistency would probably be better here. /cc @FrankHassanabad

Reply:
Consistency would be better. We should consolidate on `signals index`. `spaces index` is more confusing. You are correct. Everyone understands a signals index but won't understand what a spaces index means.